pkg/config/system_probe.go

```diff
@@ -210,7 +208,8 @@ func InitSystemProbeConfig(cfg Config) {
 	cfg.BindEnvAndSetDefault(join(smNS, "enable_http_stats_by_status_code"), false)
 
 	cfg.BindEnvAndSetDefault(join(netNS, "enable_gateway_lookup"), true, "DD_SYSTEM_PROBE_NETWORK_ENABLE_GATEWAY_LOOKUP")
-	cfg.BindEnvAndSetDefault(join(netNS, "max_http_stats_buffered"), 100000, "DD_SYSTEM_PROBE_NETWORK_MAX_HTTP_STATS_BUFFERED")
+	// Default value (100000) is set in `adjustUSM`, to avoid having "deprecation warning", due to the default value.
```

Comment:
Beside the "deprecation warning" issue, if we set a default value we won't know what value was input by the user, so we'll have hard time deciding which value should be taken. This is the reason the value `enable_http_monitoring` doesn't have this comment

Reply:
sorted the issue by heart